number.js

```diff
@@ -0,0 +1,2 @@
+example = 123456789;
```

Comment:
always `var` the variables. this would be defining `window.example` which in most cases you won't want.